### PR TITLE
Tweaking of Pharmaceuticals text for resubmission

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

Looking through the text, it seemed like the structure was still a bit unclear. I really hope this is a step in the right direction. I tried to split it into these major categories:
1. Adapt clinical strategies from other outbreaks to repurpose drugs for symptom management
2. Adapt from virological research to repurpose antivirals
3. Adapt from immunological research to repurpose drugs that either stop the virus from overwhelming cells, or else bolster the immune response
4. Using high-throughput screening can be an alternate way to approach this for any dimension

The appendix has also been edited to remove redundancies

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/20.pharmaceuticals.md [hunk 1]
```diff
@@ -27,54 +27,56 @@ The novel coronavirus _Severe acute respiratory syndrome-related coronavirus 2_
...
 
-### Potential Approaches to the Treatment of COVID-19
+#### Potential Approaches to the Treatment of COVID-19
 
-Today, many different approaches are available for identifying existing pharmaceuticals or candidate compounds that might be useful in mitigating the effects of a viral infection.
-Two common therapeutic strategies are to either reduce the symptoms that are harmful to patients or hinder the spread of infection by targeting a virus directly.
-The goal of the former is to reduce the severity and risks of an active infection, while for the latter, it is to inhibit the replication of the virus once an individual is infected, potentially freezing disease progression.
-The identification of potential therapeutics against an emerging disease or a novel virus can proceed through either drug repurposing or drug development.
+Therapeutic interventions can utilize two approaches: they can either mitigate the effects of an infection that harm an infected person, orthey can hinder the spread of infection within a host by disrupting the viral life cycle.
```
Comment (cgreene) [inline content/20.pharmaceuticals.md:77]:
```suggestion
Therapeutic interventions can utilize two approaches: they can either mitigate the effects of an infection that harm an infected person, or they can hinder the spread of infection within a host by disrupting the viral life cycle.
```

File: content/20.pharmaceuticals.md [hunk 2]
```diff
@@ -85,86 +87,78 @@ The rapid timescale of this response meant that, initially, most evidence came f
...
-Broad-spectrum treatments are not specifically targeted at a virus or at particular host receptors, but rather induce broad shifts in host biology that are hypothesized to be potential inhibitors of the virus.
-Instead, when a virus enters a host, the host becomes the virus's environment, and, therefore, the state of the host can also influence the virus's ability to replicate and spread.
-The administration and assessment of such treatments on a rapid time course was feasible because they are often either available in hospitals, or in some cases may also be prescribed to a large number of out-patients.
-One of the other advantages is that these well-established compounds, if found to be beneficial, tend to be more widely available than boutique experimental drugs.
+
+Many treatments considered for COVID-19 have relied on a broad-spectrum approach.
+Such treatments do not specifically target a virus or particular host receptor, but rather induce broad shifts in host biology that are hypothesized to be potential inhibitors of the virus.
```
Comment (cgreene) [inline content/20.pharmaceuticals.md:130]: Is this specifically in the antiviral space (as written, I think it is)? It feels like the immunomodulatory treatments (especially steroids) are the quintessential broad-spectrum treatment.

File: content/20.pharmaceuticals.md [hunk 3]
```diff
@@ -423,36 +432,29 @@ Other large-scale efforts to screen candidate drugs against SARS-CoV-2 (e.g., [@
...
-![
-**Mechanism of Action for Potential Therapeutics**
-Potential therapeutics currently being studied can target the SARS-CoV-2 virus or modify the host environment through many different mechanisms.
-Here, the relationship between the virus and several therapeutics described above are visualized.
-](images/N001-LifeCyclePlusDrugs.png){#fig:therapeutics secno=1}
-
-Two approaches seek to identify candidate pharmaceuticals based on information gained in non-clinical research on HCoV.
-The second approach reviewed above uses information about how related viruses function to identify possible ways to disrupt pathogenesis.
-These compounds are also typically small-molecule drugs, but often includes boutique and experimental medications like remdesivir,
+Over the last year, dexamethasone has emerged as the strongest treatment against severe COVID-19, but other clinically suggested compounds such as HCQ/CQ and ARBs/ACEis have not borne out early hope in their mechanistic relevance.
```
Comment (cgreene) [inline content/20.pharmaceuticals.md:439]: Small molecule treatment? Would vaccines be included here?

**Review by cgreene — APPROVED** [review]
I had a few questions / small revisions but I think that with discretionary changes this is good to go 🚀

File: content/20.pharmaceuticals.md (hunk 2 above)
Comment (rando2) [reply content/20.pharmaceuticals.md:130]: Good point! Something definitely got lost here in the rearranging.
```suggestion
In addition to broad-spectrum treatments like dexamethasone, antivirals offer an alternative approach.
These treatments do not specifically target a virus or particular host receptor, but rather induce broad shifts in host biology that are hypothesized to be potential inhibitors of the virus.
```

File: content/20.pharmaceuticals.md (hunk 3 above)
Comment (rando2) [reply content/20.pharmaceuticals.md:439]: I thought the vaccines are biologics -- but I was only in the pharmacology department for eight months, so will definitely defer to you 😆  Regardless, we should probably mention vaccines in this discussion...
```suggestion
Over the last year, dexamethasone has emerged as the strongest treatment against severe COVID-19, but other clinically suggested compounds such as HCQ/CQ and ARBs/ACEis have not borne out early hope in their mechanistic relevance.
A number of vaccine development programs have also met huge success using a range of strategies [@individual-vaccines].
```

Comment (cgreene) [reply content/20.pharmaceuticals.md:439]: It's up to you - I just wasn't sure if we were grouping or distinguishing between them. I'd consider this optional :)

File: content/20.pharmaceuticals.md (hunk 2 above)
Comment (cgreene) [reply content/20.pharmaceuticals.md:130]: 👍

File: content/20.pharmaceuticals.md (hunk 3 above)
Comment (rando2) [reply content/20.pharmaceuticals.md:440]: Thanks @cgreene! I'm curious what @RLordan thinks!

Comment (RLordan) [reply content/20.pharmaceuticals.md:440]: I'm just as stumped haha, but if this helps the Center for Biologics Evaluation and Research regulates childhood vaccines in the US

Comment (rando2) [reply content/20.pharmaceuticals.md:440]: Thank you @RLordan and @cgreene!

File: content/20.pharmaceuticals.md [hunk 4]
```diff
@@ -2,23 +2,23 @@
...
-However, previous insights into the immune response to coronaviruses gained during the outbreaks of severe acute respiratory syndrome (SARS) and Middle East respiratory syndrome (MERS) have proved beneficial for identifying approaches to treat and prevent novel coronavirus disease 2019 (COVID-19).
-Building on research in these and other diseases, the medical and scientific communities responded rapidly to the COVID-19 crisis and identified a large number of candidate therapeutics.
+After emerging in China in late 2019, the novel _Severe acute respiratory syndrome-like coronavirus 2_ (SARS-CoV-2) spread worldwide and as of early 2021 continues to significantly impact most countries.
+Only a small number of coronaviruses are known to infect humans, and only two cause infections similar in severity to SARS-CoV-2: _Severe acute respiratory syndrome-related coronavirus_, a closely related species of SARS-CoV-2 that emerged in 2002, and _Middle East respiratory syndrome-related coronavirus_, which emerged in 2012.
+Unlike in the current pandemic, both of these previous epidemics were controlled fairly rapidly through public health measures.
```
Comment (RLordan) [inline content/20.pharmaceuticals.md:7]:
```suggestion
Unlike in the current pandemic, both of these previous epidemics were controlled relatively rapidly through public health measures.
```

File: content/20.pharmaceuticals.md [hunk 5]
```diff
@@ -27,54 +27,56 @@ The novel coronavirus _Severe acute respiratory syndrome-related coronavirus 2_
...
-<!--To Do: Update this figure in line with #736, #962, and 10.3390/ijerph17113973 (which explains why the SARS data don't start until March)-->
 ![
-**Cumulative global COVID-19 deaths since January 22, 2020.**
-{{csse_deaths}} COVID-19 deaths had been reported worldwide as of {{csse_date_pretty}} (Figure @fig:csse-deaths).
-Data are from the COVID-19 Data Repository by the Center for Systems Science and Engineering at Johns Hopkins University [@https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series].
-Previous emergent viral threats had a much lower death toll that was not felt on a such a global scale, and in most cases were controlled much more quickly.
-For example, SARS-CoV-1 was controlled within 9 months of its appearance, whereas SARS-CoV-2 remains a significant global threat almost 1.5 years later.
+**Cumulative global incidence of COVID-19 and SARS.**
+As of {{csse_date_pretty}}, {{csse_cases}} COVID-19 cases and {{csse_deaths}} COVID-19 deaths had been reported worldwide since January 22, 2020 (Figure @fig:csse-deaths).
+In contrast, a total of 8,432 cases and 813 deaths were reported for SARS from March 17 to July 11, 2003.
```
Comment (RLordan) [inline content/20.pharmaceuticals.md:42]: Is in contrast required here, when lines 43 and 44 are deleted?

File: content/20.pharmaceuticals.md [hunk 6]
```diff
@@ -85,86 +87,79 @@ The rapid timescale of this response meant that, initially, most evidence came f
...
 ]({{ebm_trials_figure}} "COVID-19 clinical trials"){#fig:ebm-trials secno=1}
 
 The purpose of this review is to provide an evolving resource tracking the status of efforts to repurpose and develop drugs for the treatment of COVID-19.
 We highlight four strategies that provide different paradigms for the identification of potential pharmaceutical treatments.
 As results become available from additional clinical trials, we will continue to update this manuscript to keep pace with the current understanding of which therapeutics may be effective against SARS-CoV-2 or COVID-19.
 
-### Clinical Response and Small Molecule Drug Repurposing
+### Repurposing Drugs for Symptom Management
 
 A variety of symptom profiles with a range of severity are associated with COVID-19.
```
Comment (RLordan) [inline content/20.pharmaceuticals.md:111]: Can probably cite the pathogenesis manuscript there

**Review by RLordan — APPROVED** [review]

File: content/20.pharmaceuticals.md (hunk 5 above)
Comment (rando2) [reply content/20.pharmaceuticals.md:42]:
```suggestion
A total of 8,432 cases and 813 deaths were reported for SARS from March 17 to July 11, 2003.
```

**Pull request merged**